modules/apps/jaggery/emm/src/emm/units/platform-configuration/public/js/platform-configuration.js

```diff
@@ -28,6 +28,21 @@ function inputIsValid(regExp, inputString) {
 }
 
 /**
+ * Get valid certificate.
+ *
+ * @param  certificate
+ * @param cached certificate (in the registry)
+ * @returns {String} Returns the valid certificate
+ */
+function getCertificate(certificate, cachedCertificate) {
```

Comment:
Change method name and the parameter names according to the below comment (L621)